flow-decomposition/src/main/java/com/powsybl/flow_decomposition/LoadFlowRunningService.java

```diff
@@ -32,22 +32,22 @@ class LoadFlowRunningService {
     Result runAcLoadflow(Network network, LoadFlowParameters loadFlowParameters, boolean isDcFallbackEnabledAfterAcDivergence) {
         LoadFlowParameters acEnforcedParameters = enforceAcLoadFlowCalculation(loadFlowParameters);
         LoadFlowResult acLoadFlowResult = runner.run(network, acEnforcedParameters);
-        if (!acLoadFlowResult.isOk() && isDcFallbackEnabledAfterAcDivergence) {
+        if (!acLoadFlowResult.isFullyConverged() && isDcFallbackEnabledAfterAcDivergence) {
             LOGGER.warn("AC loadflow divergence. Running DC loadflow as fallback procedure.");
             return runDcLoadflow(network, loadFlowParameters)
                 .setFallbackHasBeenActivated(FALLBACK_HAS_BEEN_ACTIVATED);
         }
-        if (!acLoadFlowResult.isOk()) {
-            throw new PowsyblException("AC loadfow divergence without fallback procedure enabled.");
+        if (!acLoadFlowResult.isFullyConverged()) {
+            throw new PowsyblException("AC loadflow divergence without fallback procedure enabled.");
```

Comment:
Well spotted !
We have seen some `PDTF` instead of `PTDF` in the observer code as well